sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -230,13 +246,15 @@ func runCtr(containerdSocket string, namespace string, containerID string, sourc
 				// Mount in the persistent storage location for this container
 				{
 					Options:     []string{"rbind", "rw"},
-					Destination: "/.bottlerocket/host-containers/" + containerID,
-					Source:      "/local/host-containers/" + containerID,
+					Destination: "/.bottlerocket/" + persistentPath,
+					Source:      "/local/" + persistentPath,
 				}}),
 			// Mount the rootfs with an SELinux label that makes it writable
 			withMountLabel("system_u:object_r:state_t:s0"),
 			// Include conditional options for superpowered containers.
 			withSuperpowered(superpowered),
+			// Include conditional options for bootstrap or superpowered containers
+			withPrivilegedAccess(superpowered || bootstrap),
```

Comment:
Will `withSuperpowered` ever be called without `withPrivilegedAccess`?  The refactoring here moves some of the critical pieces of superpowered out into "privileged" that if you called `withSuperpowered` alone you wouldn't end up with a functional superpowered container.

Reply:
I removed the `withPrivilegedAccess` method since bootstrap containers shouldn't have all privileges just yet.